README.md

```diff
@@ -51,20 +46,11 @@ such as the `entrypoint.sh`. Then the subfolders organize the app files:
 
 1. Go to docker hub to get the latest patch version for
     the [python docker image](https://hub.docker.com/_/python/)
-2. Then 3 files need to be edited with the latest python minor or patch versions:
-    1. `docker-compose.generate_requirements.yml` to generate the requirements files.
+2. Then 2 files need to be edited with the latest python minor or patch versions:
     1. `.github/workflows/docker_image.yml` to build and push the right images to docker hub.
     1. `Dockerfile` to change the default base image version. This is used only for testing
```

Comment:
Should probably mention which Dockerfile, as we have two in the repo, just to be super clear

Reply:
Good point. Done.